src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");

```

Comment:
The message 'Truck started its work' could be more informative. It should specify what kind of truck is working, for example, 'Dump truck started its work', to provide clearer information about the action taking place.